### PR TITLE
Allow Exec Plugin Use with k8s provider, based on provider 1.10

### PR DESCRIPTION
**Pull request by liskl**
ok another pass at reducing changes.
I swapped in the changes from terraform-kubernetes-provider v1.10.0
```
liskl/original└─ $ make test
==> Checking that code complies with gofmt requirements...
go test -i $(go list ./... |grep -v 'vendor') || exit 1
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
echo $(go list ./... |grep -v 'vendor') | \
		xargs -t -n4 go test  -timeout=30s -parallel=4
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
go test -timeout=30s -parallel=4 github.com/nabancard/terraform-provider-kubernetes-yaml github.com/nabancard/terraform-provider-kubernetes-yaml/kubernetes
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
?   	github.com/nabancard/terraform-provider-kubernetes-yaml	[no test files]
ok  	github.com/nabancard/terraform-provider-kubernetes-yaml/kubernetes	1.005s
```

```
liskl @ ~/terraform-provider-kubernetes-yaml
liskl/original└─ $ make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./kubernetes -v  -timeout 120m -count=1
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
=== RUN   TestPartialCompare
=== RUN   TestPartialCompare/Simple_map_with_string_value
=== RUN   TestPartialCompare/Simple_map_with_string_value_and_Skippable_fields
2020/02/14 14:28:56 [COMPARE] Skipping as in SkipFields: "1245" "1245"
=== RUN   TestPartialCompare/Map_with_nested_map[string]string
=== RUN   TestPartialCompare/Map_with_nested_map[string]string_with_different_ordering
=== RUN   TestPartialCompare/Map_with_nested_map[string]string_with_nested_array
=== RUN   TestPartialCompare/Map_with_nested_map[string]string_with_nested_array_and_nested_map
=== RUN   TestPartialCompare/Different_Ordering
=== RUN   TestPartialCompare/Nested_Array
=== RUN   TestPartialCompare/Ignore_additional_fields
=== RUN   TestPartialCompare/Handle_removed_fields
=== RUN   TestPartialCompare/Handle_integers
=== RUN   TestPartialCompare/Handle_updated_field._Expect_returned_value_to_be_shown
=== RUN   TestPartialCompare/Map_with_nested_map[string]string_with_updated_field
=== RUN   TestPartialCompare/Handle_duplicate_name_fields_in_nested_maps
--- PASS: TestPartialCompare (0.00s)
    --- PASS: TestPartialCompare/Simple_map_with_string_value (0.00s)
    --- PASS: TestPartialCompare/Simple_map_with_string_value_and_Skippable_fields (0.00s)
    --- PASS: TestPartialCompare/Map_with_nested_map[string]string (0.00s)
    --- PASS: TestPartialCompare/Map_with_nested_map[string]string_with_different_ordering (0.00s)
    --- PASS: TestPartialCompare/Map_with_nested_map[string]string_with_nested_array (0.00s)
    --- PASS: TestPartialCompare/Map_with_nested_map[string]string_with_nested_array_and_nested_map (0.00s)
    --- PASS: TestPartialCompare/Different_Ordering (0.00s)
    --- PASS: TestPartialCompare/Nested_Array (0.00s)
    --- PASS: TestPartialCompare/Ignore_additional_fields (0.00s)
    --- PASS: TestPartialCompare/Handle_removed_fields (0.00s)
    --- PASS: TestPartialCompare/Handle_integers (0.00s)
    --- PASS: TestPartialCompare/Handle_updated_field._Expect_returned_value_to_be_shown (0.00s)
    --- PASS: TestPartialCompare/Map_with_nested_map[string]string_with_updated_field (0.00s)
    --- PASS: TestPartialCompare/Handle_duplicate_name_fields_in_nested_maps (0.00s)
=== RUN   TestDiffStringMap
=== RUN   TestDiffStringMap/0
=== RUN   TestDiffStringMap/1
=== RUN   TestDiffStringMap/2
=== RUN   TestDiffStringMap/3
=== RUN   TestDiffStringMap/4
=== RUN   TestDiffStringMap/5
=== RUN   TestDiffStringMap/6
--- PASS: TestDiffStringMap (0.00s)
    --- PASS: TestDiffStringMap/0 (0.00s)
    --- PASS: TestDiffStringMap/1 (0.00s)
    --- PASS: TestDiffStringMap/2 (0.00s)
    --- PASS: TestDiffStringMap/3 (0.00s)
    --- PASS: TestDiffStringMap/4 (0.00s)
    --- PASS: TestDiffStringMap/5 (0.00s)
    --- PASS: TestDiffStringMap/6 (0.00s)
=== RUN   TestEscapeJsonPointer
--- PASS: TestEscapeJsonPointer (0.00s)
=== RUN   TestProvider
--- PASS: TestProvider (0.00s)
=== RUN   TestAccK8srawYaml
=== RUN   TestAccK8srawYaml/File:_../_examples/crds/basic_crd.tf
=== RUN   TestAccK8srawYaml/File:_../_examples/crds/couchbase.tf
=== RUN   TestAccK8srawYaml/File:_../_examples/ingress_basic/basic_ingress.tf
=== RUN   TestAccK8srawYaml/File:_../_examples/ingress_complex/complex_ingress.tf
=== RUN   TestAccK8srawYaml/File:_../_examples/roles/clusterrole.tf
=== RUN   TestAccK8srawYaml/File:_../_examples/service/basic_service.tf
--- PASS: TestAccK8srawYaml (18.38s)
    --- PASS: TestAccK8srawYaml/File:_../_examples/crds/basic_crd.tf (5.04s)
    --- PASS: TestAccK8srawYaml/File:_../_examples/crds/couchbase.tf (4.13s)
    --- PASS: TestAccK8srawYaml/File:_../_examples/ingress_basic/basic_ingress.tf (2.17s)
    --- PASS: TestAccK8srawYaml/File:_../_examples/ingress_complex/complex_ingress.tf (1.88s)
    --- PASS: TestAccK8srawYaml/File:_../_examples/roles/clusterrole.tf (3.43s)
    --- PASS: TestAccK8srawYaml/File:_../_examples/service/basic_service.tf (1.72s)
PASS
ok  	github.com/nabancard/terraform-provider-kubernetes-yaml/kubernetes	19.353s
```

```
liskl @ ~/terraform-provider-kubernetes-yaml
liskl/original└─ $ git log -n1
6ecc8c6 (HEAD -> liskl/original, origin/liskl/original) moved back to provider 1.10 as the source.
```

```
liskl @ ~/terraform-provider-kubernetes-yaml
liskl/original└─ $ make build
==> Checking that code complies with gofmt requirements...
go install
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
```

```
liskl @ ~/terraform-provider-kubernetes-yaml
liskl/original└─ $ make build-binaries
Building for linux amd64...
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
Building for windows amd64...
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
Building for windows 386...
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
Building for darwin amd64...
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
Building for linux 386...
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
Building for linux arm...
go: finding k8s.io/client-go v12.0.0+incompatible
go: finding k8s.io/client-go v12.0.0+incompatible
Completed builds, for output see ./bin
liskl @ ~/terraform-provider-kubernetes-yaml
```

### CODE REVIEW

**Comment by liskl** [comment]
this is way closer to your initial codebase so as to not cause any huge differences.

**Comment by davisford** [comment]
Hi @liskl I am sorry for the serious lag in response -- I have github notifications going to a folder where I forget to check.  You have two PRs here.  They seem similar.  Are they both valid?  I will try to review and get approved and merged here, but just want to make sure I understand if both are valid?

File: go.mod [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+go 1.13
 
 require (
+	cloud.google.com/go/bigtable v1.3.0 // indirect
 	github.com/cenkalti/backoff v2.1.1+incompatible
-	github.com/google/btree v1.0.0 // indirect
-	github.com/hashicorp/terraform v0.12.0
-	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
+	github.com/dustinkirkland/golang-petname v0.0.0-20191129215211-8e5a1ed0cff0 // indirect
+	github.com/hashicorp/terraform v0.11.11 // indirect
```
Comment (davisford) [inline go.mod:9]: Why is Terraform module being rolled back here from `v0.12.0` to `v0.11.11` ?

Comment (liskl) [reply go.mod:9]: Sorry we can close this one in favor of the other which is a better and cleaner  patch.

**Pull request closed without merging**